app/scripts/directives/processTemplateDialog.js

```diff
@@ -124,5 +234,82 @@
     function instantiateTemplate() {
       $scope.$broadcast('instantiateTemplate');
     }
+
+    function filterForKeywords (searchText, items) {
+      return KeywordService.filterForKeywords(items, ['name', 'tags'], KeywordService.generateKeywords(searchText));
+    }
+
+    function filterChange (filters) {
```

Comment:
Tiny nit: some inconsistent parens spacing in this file `function filterChange (filters) ` vs `function filterItems()`

Reply:
Fixed